homeassistant/components/elmax/common.py

```diff
@@ -71,6 +72,12 @@ def get_zone_state(self, zone_id: str) -> Actuator:
             return self._state_by_endpoint.get(zone_id)
         raise HomeAssistantError("Unknown zone")
 
+    def get_area_state(self, area_id: str) -> Area:
+        """Return state of a specific area."""
+        if self._state_by_endpoint is not None:
+            return self._state_by_endpoint.get(area_id)
```

Comment:
Could this return `None` (the default of `get`)? If so, we should update the method's return typing.

Reply:
Good point. It shall never return None, therefore I've changed the way I access the state dict: using the squared-bracket operator instead of GET(). That makes the type hint consistent.
Nice spot!

Addressed via 8c6fc786257f6b18380ffc5f60d7900abf42792e